README.md

```diff
@@ -1 +1,146 @@
-# mobile-menu-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Mobile-menu-project <a name="about-project"></a>
+
+**Mobile Menu Project** 
+Created a mobile menu for the portfolio project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Client</summary>
+  <ul>
+    <li><a href="https://www.html.org/">HTML</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://www.css.org/">CSS</a></li>
+  </ul>
+      <ul>
+    <li><a href="https://www.figma.org/">Figma</a></li>
+  </ul>
+    </ul>
+      <ul>
+    <li><a href="https://www.javascript.org/">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [View Project](https://nahnahaj.github.io/mobile-menu-project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+- Have a Desktop or Laptop Computer running on with Windows, Mac OS or Linux operating system.
+- Have git installed on the Computer
+- Have a browser install on the Computer
+
+### Setup
+
+- If it suits you, on your local computer, create a directory in your preferred location (e.g in the Desktop or Document directory)
+
+### Install
+
+- Open your Terminal/Powershell or any command line interface you use, then navigate to your created above directory or the directory you'll like to clone this project directory.
+- Type `git clone git clone https://github.com/NahnahAJ/mobile-menu-project.git` if your're using https with git or `git@github.com:NahnahAJ/mobile-menu-project.git` if your preferred is ssh and hit enter.
+- Navigate to the `create-mobile-menu` directory on your local computer in your above chosen directory and open the `index.html` file.
+
+### Usage
+
+- You can view the HTML document in your preferred or default browser.  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Nahnah Ahjuwah**
+
+- GitHub: [@NahnahAJ](https://github.com/NahnahAJ)
+- Twitter: [@NahnahAhjuwah](https://https://twitter.com/NahnahAhjuwah)
```

Comment:
[OPTIONAL] 
This twitter link is broken and not working. I think fixing it would make it easy for hiring managers, recruiters, and other professionals to reach out to you.